opensrp-unicef-tunisia/build.gradle

```diff
@@ -72,6 +70,25 @@ android {
         vectorDrawables.useSupportLibrary = true
     }
 
+    applicationVariants.all { variant ->
+        if (variant.name == "debug") {
+            variant.outputs.each { output ->
+                output.versionCodeOverride = 4
+                output.versionNameOverride = "0.2.0-debug"
+            }
+        } else if (variant.name == "preview") {
+            variant.outputs.each { output ->
+                output.versionCodeOverride = 4
+                output.versionNameOverride = "0.2.0-preview"
+            }
+        } else if (variant.name == "release") {
+            variant.outputs.each { output ->
+                output.versionCodeOverride = 4
+                output.versionNameOverride = "0.2.0-release"
```

Comment:
Reset this version code to 1 and v0.0.1 as we have not had any release yet

Reply:
Done